src/components/displayQoute.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react"
+
+function Qoute() {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
Great job on the implementation 👍 . According to the requirement, there should be an error state, kindly implement it. 🥺